services/slam/fake/slam_test.go

```diff
@@ -19,16 +19,15 @@ func TestFakeSLAMPosition(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, pInFrame.Parent(), test.ShouldEqual, slamSvc.Name)
 
-	// These test ranges as I observed different floating point values
-	// between M1 mac hosts and amd64 hosts for the same test data.
-	test.That(t, pInFrame.Pose().Point().X, test.ShouldBeBetween, 1.860775180178, 1.860775180179)
-	test.That(t, pInFrame.Pose().Point().Y, test.ShouldBeBetween, 34.26593374183, 34.26593374184)
+	// test.ShouldBeBetween is used here as tiny differences were observed
+	// in floating point values between M1 mac & arm64 linux which
+	// were causing tests to pass on M1 mac but fail on ci.
+	test.That(t, pInFrame.Pose().Point().X, test.ShouldBeBetween, -0.005885172861759, -0.005885172861758)
```

Comment:
[q] Normally I'd recommend we loop through the saved data to confirm the output data matches the saved data, but this might be overkill considering we'd just be replicating the function's internals to do so. But please confirm

Reply:
In my opinion, given that this is a test dataset and we are going to want to make it easy & ergonomic to update this to new datasets in the future, I'm inclined to leave this test for a single example. 

Otherwise, there will be a high burden to anyone who wants to change fake SLAM to use a new dataset. 

I'm confident that TestFakeSLAMStateful https://github.com/viamrobotics/rdk/blob/6ecea44b0f4877c0d6e7ee4f727e8761fd182869/services/slam/fake/slam_test.go#L142 proves that the test data is changing on every call to GetMap & is looping back around. I think that gets us 90% of the value of verifying that each call increments the test data by 1 without needing to re write all the tests every time we change the test data.